live/app/components/qroc-solution-panel.js

```diff
@@ -28,11 +28,8 @@ const QrocSolutionPanel = Ember.Component.extend({
     if (!solutionVariants){
```

Comment:
Petit refacto au passage :
-------------------------
const resultValue = {
  	ok: 'correction-qroc-box__input-right-answer',
  	ko: 'correction-qroc-box__input-wrong-answer',
  	aband: 'orrection-qroc-box__input-no-answer'
  }

  inputClass : Ember.computed('answer.result', function(){
  	return resultValue[this.get('answer.result')] || '';
  });